test/es-module/test-esm-loader-mock.mjs

```diff
@@ -1,9 +1,6 @@
-// Flags: --loader ./test/fixtures/es-module-loaders/mock-loader.mjs
-import '../common/index.mjs';
-import assert from 'assert/strict';
-
-// This is provided by test/fixtures/es-module-loaders/mock-loader.mjs
-import mock from 'node:mock';
+// Flags: --import ./test/fixtures/es-module-loaders/mock.mjs
```

Comment:
nit: we could simply remove this since we import `mock.mjs` below.
```suggestion
```

Reply:
I'd rather not because this is more typical usage, and it ensures correct load order.

Reply:
I'd argue this is not typical usage – most folks would rather just `import 'my-awesome-mock-lib'` than use a CLI flag.